src/main/java/Objects/Packets/Packet.java

```diff
@@ -1,10 +1,78 @@
 package Objects.Packets;
 
+/**
+ * Parent class that all specific
+ * types of packets extends.
+ */
 public class Packet {
 
-    private int id; //TODO: necessary?
-    private int team;
-    private int destination;
-    //TODO: more?
+    private String team;
+    private int cost;
+    private int stealth;
+    private String packet_type;
```

Comment:
is this really needed? wont the class be enough?

Reply:
Changed to "return getClass()"
Will need to figure out implementation for that for interaction with devices.

Reply:
We don't need the variable at all was the point

Reply:
The function that returns class is pointless as well